.eslintrc.js

```diff
@@ -125,5 +122,12 @@ module.exports = {
         'import/no-extraneous-dependencies': 'off',
       },
     },
+    {
+      // Example files and codemods are allowed to log to console
+      files: ['**/examples/**', '**/packages/forma-36-codemod/**/*'],
+      rules: {
+        'no-console': 'off',
```

Comment:
Should we go with warn? So at least VSCode can highlight it

Reply:
It's not disabling the rule globally, just for our code examples — where we're using it in click handlers — and our codemod, which is supposed to write to the console 🙂 It's still a warning everywhere else.